README.md

```diff
@@ -129,15 +146,30 @@ This allows the checking of commits for a PR and merging them when the right lab
 
 `VersionBot` will ignore any status checks and not attempt to merge should the `procbots/versionbot/no-checks` label be present on any PR it would otherwise operate on when a PR is opened.
 
-Current, `VersionBot` will:
+Currently, `VersionBot` will:
 * Version up only once all status checks set as 'Required' on protected branches for 'master' are successful
-* The `procbots/versionbot/ready-to-merge` is present
+* The `procbots/versionbot/ready-to-merge` label is present
+
+VersionBot can be configured via `.procbots.yml` file present in any repo that it operates on. This alters the settings for it for working on that repository. Currently the configuration for VersionBot consists of the following properties:
+
+    `procbot.githubbot.versionbot.maintainers` - A list of Github user names that are authorised maintainers of the repository. The `procbots/versionbot/ready-to-merge` label will only be acted upon should a maintainer in this list have added the label.
+
+Example:
+
+```
+procbot:
+    minimum_version: 0.5
+    githubbot:
+        versionbot:
```

Comment:
Yes, I'd quite like to keep this for exactly that reason, especially if there's a test in the `githubbot` block that causes it to fail for all children. It's immediately obvious then when adding/modifying properties.